.gitignore

```diff
@@ -1,10 +1,12 @@
 bin/
 obj/
+run/
```

Comment:
Sorry for hustle, I know that I suggested this name `run` yesterday. I didn't find a good one at the moment, but now I think we should use `bin` instead of `run`, because it would match dotnet output.